src/main/java/com/cdancy/jenkins/rest/JenkinsUtils.java

```diff
@@ -288,6 +291,36 @@ public static void setEnvironmentVariables(final Map<String, String> newEnvVars)
         }
     }
 
+    /**
+     * amend the path to job by appending "/job/" before every folder name
+     *
+     * @param folderPath simple folder path expression
+     *
+     * @return amendedFodlerPath which optional folder
+     *         path reconstructed to match the jenkins URL style.
+     */
+    public static String amendFolderPath(String folderPath) {
+        Pattern pattern = Pattern.compile("(job/[^!@#$%^&*<>]+/*)+");
+        Matcher m = pattern.matcher(folderPath);
+        if(m.matches()) {
+            return folderPath;
+        }
+
+        if(folderPath.startsWith("/") || folderPath.endsWith("/")) {
+            throw new RuntimeException("Incorrect folder Path format - " + folderPath);
+        }
+
+        String amendedPath = "";
+        String[] folderNames = folderPath.split("/");
+        for (int i=0; i<folderNames.length; i++) {
+            amendedPath += "job/" + folderNames[i];
+            if(i < folderNames.length-1) {
+                amendedPath += "/";
+            }
+        }
```

Comment:
I don't want to add another library dependency for just one method if it can be helped. Instead we can recreate the method in our utilities file if need be.